bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Component.java

```diff
@@ -72,6 +72,6 @@ default ComponentData getData() {
     @NotNull
     @Override
     default String getExportedType() {
```

Comment:
can't this method not also be removed because its an override ?

Reply:
It overrides the method in `ComponentExporter` which does not provide a `default`.